doc/api_blueprints/snippet-sharing.apib

```diff
@@ -1,4 +1,5 @@
 FORMAT: 1A
+HOST: api.libelektra.org
```

Comment:
I only added it as placeholder for the moment. I'll have to change it to some localhost url for testing, that's why I needed it. We can change it when the API and the deployment decisions are final.